frontend-next/api/posts/comments/actions.ts

```diff
@@ -0,0 +1,31 @@
+import { CreateCommentType } from '@models/comment';
+import { deleteRequest, post, put } from 'utils/request';
+
+export function createComment(commentBody: CreateCommentType, postId: string) {
+  return new Promise((resolve: (response: any) => void, reject) =>
+    post(`/api/v2/posts/${postId}/comments`,
+          commentBody,
+    ).then(response => {
+      resolve(response);
+    }).catch(error => {
+      reject(error);
+    })
+  );
+};
+
+export function onLikedComment(postId: string, commentId: string) {
+  return new Promise((resolve: (response: any) => void, reject) =>
+    put(`/api/v2/posts/${postId}/comments/${commentId}/like`, {})
+      .then(response => {
+        resolve(response);
+      }).catch(error => {
+        reject(error);
+      })
+  );
+};
```

Comment:
This Promise constructor is redundant.

```suggestion
export function onLikedComment(postId: string, commentId: string) {
  return put(`/api/v2/posts/${postId}/comments/${commentId}/like`, {});
};
```